internal/processorinfo/processor_count.go

```diff
@@ -0,0 +1,29 @@
+package processorinfo
+
+import (
+	"runtime"
+	"syscall"
+)
+
+var (
+	modkernel32 = syscall.NewLazyDLL("kernel32.dll")
+	// https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getmaximumprocessorcount
+	getMaxProcCount = modkernel32.NewProc("GetMaximumProcessorCount")
```

Comment:
Can you double check what the behavior of `GetMaximumProcessorCount` will be in a minroot configuration? My assumption is that it will only see the processors allocated to the root partition.

Reply:
This is only called to normalize the processor count for process isolated containers. For normalizing the UVMs amount the HCS apis are used.

Reply:
Okay cool, I think I misunderstood the comment.